packages/core/src/store/level-db-store.ts

```diff
@@ -86,12 +86,23 @@ export class LevelDbStore implements IKVStore {
 
   async del(key: string, useCaseName?: string): Promise<void> {
     const store = await this.#storeMap.get(useCaseName)
-    return await store.del(key)
+    try {
+      return await store.del(key)
+    } catch (err) {
+      const msg = `Error deleting key ${key} from leveldb state store: ${err}`
+      this.logger.warn(msg)
+      throw new Error(msg)
+    }
   }
 
   async get(key: string, useCaseName?: string): Promise<any> {
     const store = await this.#storeMap.get(useCaseName)
-    return await store.get(key)
+    try {
+      return await store.get(key)
+    } catch (err) {
+      this.logger.warn(`Error fetching key ${key} from leveldb state store: ${err}`)
+      throw err
```

Comment:
Should we align this with the other two changes in this file?

Reply:
yeah, it's tricky because higher levels rely on the Error type thrown by the underlying leveldb library.  I think I may have a workaround though, testing it now

Reply:
okay, updated, PTAL